plinth/views.py

```diff
@@ -30,8 +30,28 @@
 
 from . import forms, frontpage
 import plinth
+from django.utils.translation import ugettext_lazy as _
 
+from plinth import cfg
 
+def apps_init():
+    """Initailize the apps module"""
+    cfg.main_menu.add_urlname(_('Apps'), 'glyphicon-download-alt',
+                              'apps:index')
+
+def sys_init():
+    """Initialize the system module"""
+    cfg.main_menu.add_urlname(title, 'glyphicon-cog', 'system:index')
+
+def apps_index(request):
+    """Serve the apps index page"""
+    return TemplateResponse(request, 'apps.html', {'title': _('Applications')})
+
+def sys_index(request):
+    """Serve the system index page"""
+    return TemplateResponse(request, 'system.html',
+                            {'title': system.title,
```

Comment:
system.title and system.description won't work here if the system module is removed.